### PR TITLE
docs(forms): fix reactive forms api examples

### PR DESCRIPTION
**Pull request by kara**
This PR updates the old docs examples in reactive form directives.

### CODE REVIEW

File: modules/@angular/forms/src/directives/reactive_directives/form_group_directive.ts [hunk 1]
```diff
@@ -60,37 +58,35 @@ export const formDirectiveProvider: any = {
...
  *      })
  * class LoginComp {
- *  credentials: {login: string, password: string};
  *  loginForm: FormGroup;
  *
  *  constructor() {
  *    this.loginForm = new FormGroup({
- *      login: new FormControl(""),
- *      password: new FormControl("")
+ *      login: new FormControl(''),
```
Comment (vsavkin) [inline modules/@angular/forms/src/directives/reactive_directives/form_group_directive.ts]: I think double quotes are superior :)

**Comment by kara** [comment]
verbal lgtm from victor

File: modules/@angular/forms/src/directives/reactive_directives/form_group_directive.ts (hunk 1 above)
Comment (kara) [inline modules/@angular/forms/src/directives/reactive_directives/form_group_directive.ts]: Im not that opinionated, but wanted to make it consistent with the other FormControl examples.

**Comment by vikerman** [comment]
TAP Green

**Pull request merged**